backend/compact-connect/lambdas/license-data/handlers/bulk_upload.py

```diff
@@ -41,7 +41,9 @@ def _bulk_upload_url_handler(event: dict, context: LambdaContext):  # pylint: di
     upload = config.s3_client.generate_presigned_post(
         Bucket=config.bulk_bucket_name,
         Key=f'{compact}/{jurisdiction}/{uuid4().hex}',
-        ExpiresIn=config.presigned_post_ttl_seconds
+        ExpiresIn=config.presigned_post_ttl_seconds,
+        # Limit content length to ~30MB, ~200k licenses
```

Comment:
Yep. The big thing I wanted to find out with the load test was how many records I could reliably ingest in a single file. With the included tuning, I got it up to ~260k licenses. I dialed that back a bit to provide some margin, then set a limit on the file size, so that we are somewhat protected from the system getting overwhelmed. My understanding of the boards so far is that ~200k licenses per upload should be enough. If not, we can revisit and further enhance our capacity.

Reply:
Got it, helpful!